### ISSUE TITLE
python Issue since 0.1.3

### ISSUE THREAD

**Issue by tikismoke** [dhmay/jamstats#41]
**Describe the bug**
Got python exception

**To Reproduce**
Linux environnement, last version. To generate a working pdf i had to rollback to version 0.1.2
I'll send you the file with facebook to test as i can't here

```
Traceback (most recent call last):
  File "/usr/local/bin/jamstats-plot-pdf", line 35, in <module>
    logging.basicConfig(level=logging.WARNING, format=formatter)
  File "/usr/lib/python3.9/logging/__init__.py", line 2011, in basicConfig
    fmt = Formatter(fs, dfs, style)
  File "/usr/lib/python3.9/logging/__init__.py", line 570, in __init__
    self._style.validate()
  File "/usr/lib/python3.9/logging/__init__.py", line 422, in validate
    if not self.validation_pattern.search(self._fmt):
TypeError: expected string or bytes-like object

```

**Comment by dhmay** [comment]
Hm, I don't have your file yet, but I doubt it's a problem with the file. That line,

`logging.basicConfig(level=logging.WARNING, format=formatter)`

is just setting up logging. I bet 1.2 works because I hadn't added proper logging yet in 1.2. So, I suspect it has to do with your Python setup. What's your version of Python, and of the logging package? My best guess is you have a much newer or older version of logging than I do -- mine is 0.5.1.2.

**Comment by dhmay** [comment]
I did something that might fix this bug, on the main branch. I'll bundle some other stuff I'm working on now together with this and put out a new release tonight.

**Comment by dhmay** [comment]
OK, I believe this is fixed in version 0.2.0. Let me know!

**Comment by tikismoke** [comment]
I confirm it's indeed solved with 0.2.0.

Great works 👍
